cmd/root.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kf5i/k3ai-core/pkg/k8s/kctl"
+	"github.com/kf5i/k3ai-core/pkg/plugins"
+	"github.com/spf13/cobra"
+)
+
+const k3aiBinaryName = "k3ai-client"
+
+var rootCmd = &cobra.Command{
+	Use:   k3aiBinaryName,
+	Short: fmt.Sprintf(`%s installs AI tools`, k3aiBinaryName),
+	Long: fmt.Sprintf(` %s is a lightweight infrastructure-in-a-box solution specifically built to
+	install and configure AI tools and platforms in production environments on Edge
+	and IoT devices as easily as local test environments.`, k3aiBinaryName),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		pluginList, _ := plugins.GetPluginList()
```

Comment:
I would handle the error here

Reply:
This was copied verbatim from main.go. I'd also like to move commands from the root to something like `k3ai-cli apply`, `k3ai-cli delete` etc. But I don't have enough context yet to make that call.

Reply:
>This was copied verbatim from main.go.
You are right, my fault!
> I'd also like to move commands from the root to something like k3ai-cli apply, k3ai-cli delete
You are right, but let's proceed by steps